engine/rest-client/src/main/java/org/datacleaner/restclient/ComponentList.java

```diff
@@ -50,11 +51,13 @@ public void add(ComponentInfo componentInfo) {
      */
     @JsonInclude(JsonInclude.Include.NON_NULL)
     public static class ComponentInfo {
-
         private String name = "";
         private String description = "";
         private String createURL = "";
+        private Set<String> categoryNames = new HashSet<>();
+        private String superCategoryName = null;
         private Map<String, PropertyInfo> properties = new HashMap<>();
+        private byte[] iconData = null;
```

Comment:
OK that helps. I guess this is OK for now. Desktop might not always be _the_ use case but just one of many. But if the default will not include it then I think it is sensible.